Add files via upload

cojojunior opened the pull request.

cojojunior committed:
Add files via upload

brytey2k commented on index.html:
```diff
@@ -0,0 +1,54 @@
+    </div>
+
+</body>
+<script src="script.js"></script>
```
If you want to put this here, then it should be before the closing body tag.

brytey2k replied:
If you want to put this here, then it should be before the closing body tag.

cojojunior committed:
Update index.html

brytey2k commented on script.js:
I want to make sure that even when the user refreshes the page, the saved tasks will still be there. Let's find a way to include that. Let me know if you have any questions about this.

cojojunior committed:
Update script.js

adding a function whereby the user refreshes the page but saved list still shows

cojojunior committed:
Update script.js

adding automatic deletion after 30days and also keeping saved list after refreshing the page

cojojunior committed:
Update script.js

Adding saved after refreshing the site and also adding automatic deletions after 30 days

brytey2k commented on script.js:
Looking good so far
Let's add the ability mark tasks as completed.
Also let's try to encapsulate all the functions into an object.

cojojunior replied:
About encapsulating all the functions into an object, when I try there are no functionality on the page

brytey2k replied:
Oh sorry
We will go through it tomorrow

cojojunior replied:
Okay Sir

cojojunior replied:
Hello
It's being 2 weeks and I haven't heard from you please

cojojunior committed:
Update script.js

user able to mark completed task

cojojunior committed:
Update index.html

internal css removed

cojojunior committed:
Update style.css

added the removed internal css to the external css
